example/helper.rb

```diff
@@ -17,3 +17,25 @@ def info(msg)
     puts "[Stream #{@id}]: #{msg}"
   end
 end
+
+def send_promises(stream)
```

Comment:
Yes, that's true. It is only for the server.

Not sure what would be a good place to put it, unless we have a `server_helper`. Putting it at the top of `server.rb` would not be good...